godtools/App/Share/Data/MobileContentAuthTokenRepository/Cache/MobileContentAuthTokenCache.swift

```diff
@@ -10,11 +10,38 @@ import Foundation
 
 class MobileContentAuthTokenCache {
     
+    enum CacheKey: String {
+        
+        case expirationDate = "expirationDate"
+        
+        func getUserKey(userId: String) -> String {
+            return "MobileContentAuthTokenCache.\(userId).\(rawValue)"
+        }
+    }
+    
+    private let sharedUserDefaults: SharedUserDefaultsCache
+    
     let keychainAccessor: MobileContentAuthTokenKeychainAccessor
```

Comment:
Would it be worth it to separate out the user default logic since the keychain logic is separate already too?

Reply:
Hey @rachaelblue it would be cleaner.  I'm going to do that and will also look into using Realm rather than UserDefaults.